File: wiz-spring-boilerplate/src/main/java/com/example/testcodesample/domain/member/Member.java
package com.example.testcodesample.domain.member;

import org.springframework.security.crypto.password.PasswordEncoder;

import com.example.testcodesample.api.exception.BusinessException;
import com.example.testcodesample.api.exception.ErrorCode;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.AccessLevel;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Entity
@Getter
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class Member {

	@Id @GeneratedValue(strategy = GenerationType.IDENTITY)
	@Column(name = "member_id")
	private Long id;

	private String email;
	private String password;
	private String nickname;

	@Builder
	public Member(Long id, String email, String password, String nickname) {
		this.id = id;
		this.email = email;
		this.password = password;
		this.nickname = nickname;
	}
}

File: wiz-todo/src/main/kotlin/com/example/wiztodo/domain/member/entity/Member.kt
package com.example.wiztodo.domain.member.entity

import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id

@Entity
class Member(

    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null,

    val email: String,
    val password: String
) {

    fun validPassword(password: String): Boolean {
        return this.password == password
    }
}